Tests/SwiftProtobufTests/TestHelpers.swift

```diff
@@ -399,6 +399,10 @@ extension XCTestCase {
         XCTAssertTrue(actual.hasSuffix(expectedSuffix), fmt ?? "debugDescription did not match", file: file, line: line)
 #endif
     }
+    
+    func isSwiftProtobufErrorEqual(_ actual: SwiftProtobufError, _ expected: SwiftProtobufError) -> Bool {
```

Comment:
What was the verdict, is this still needed if we don't want to lock down on the message strings?

Reply:
I made the message `internal` and got rid of the `Equatable` conformance so that we don't make the message part of the public API for errors / so external callers cannot match on the message. 
However I decided to keep this assertion in unit tests just to make sure that we are surfacing the right error message in the right scenarios, because having accurate messages is still important IMO. I can get rid of this, but thought it was worth it.